stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/home/HomeFragmentViewModel.kt

```diff
@@ -0,0 +1,61 @@
+package io.getstream.chat.ui.sample.feature.home
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MediatorLiveData
+import androidx.lifecycle.ViewModel
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.livedata.ChatDomain
+import io.getstream.chat.ui.sample.common.SingleLiveEvent
+
+class HomeFragmentViewModel : ViewModel() {
+
+    private val chatDomain: ChatDomain = ChatDomain.instance()
+    private val _state: MediatorLiveData<State> = MediatorLiveData()
+    private val _events: SingleLiveEvent<UiEvent> = SingleLiveEvent()
+
+    val state: LiveData<State> = _state
+    val events: LiveData<UiEvent> = _events
+
+    init {
+        _state.value = State(
+            user = ChatClient.instance().getCurrentUser() ?: User(),
```

Comment:
Let's give the `User()` some meaningful name. e.g. `val unauthorizedUser = User("some_unique_id?")` and reuse it also in the `State.user = unauthorizedUser`. Then it would be easy to check / compare against it in other layers, like:
```kotlin
someUser == HomeFragmentViewModel.unauthorizedUser
```